src/main/groovy/com/morpheusdata/hyperv/HyperVProvisionProvider.groovy

```diff
@@ -914,6 +914,7 @@ class HyperVProvisionProvider extends AbstractProvisionProvider implements Workl
 				def hypervOpts = HypervOptsUtility.getAllHypervServerOpts(context, computeServer)
 				def stopResults = apiService.stopServer(hypervOpts, hypervOpts.name)
 				if(stopResults.success == true){
+					context.async.computeServer.updatePowerState(computeServer.id, ComputeServer.PowerState.on)
```

Comment:
the plugin service updates the power state, so this is not needed. This is also setting power state to on in the stopServer method.

Reply:
@ddeyoung changes has been implemented as suggested.